src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -737,30 +739,6 @@ bool AdaptDispatch::DeleteCharacter(const VTInt count)
     return true;
 }
 
-// Routine Description:
-// - Fills an area of the buffer with a given character and attributes.
-// Arguments:
-// - textBuffer - Target buffer to be filled.
-// - fillRect - Area of the buffer that will be affected.
-// - fillChar - Character to be written to the buffer.
-// - fillAttrs - Attributes to be written to the buffer.
-// Return Value:
-// - <none>
-void AdaptDispatch::_FillRect(TextBuffer& textBuffer, const til::rect& fillRect, const wchar_t fillChar, const TextAttribute fillAttrs)
-{
-    if (fillRect.left < fillRect.right && fillRect.top < fillRect.bottom)
-    {
-        const auto fillWidth = gsl::narrow_cast<size_t>(fillRect.right - fillRect.left);
-        const auto fillData = OutputCellIterator{ fillChar, fillAttrs, fillWidth };
-        const auto col = fillRect.left;
-        for (auto row = fillRect.top; row < fillRect.bottom; row++)
-        {
-            textBuffer.WriteLine(fillData, { col, row }, false);
-        }
-        _api.NotifyAccessibilityChange(fillRect);
```

Comment:
We don't appear to be triggering the `NotifyAccessibilityChange` event now. I believe this is needed for ConHost's accessibility support.